test/resources/stubs/testprojekt-ets6-functions.json

```diff
@@ -103,8 +103,20 @@
                             "function_type": "FT-1",
                             "project_uid": 11,
                             "group_addresses": [
-                                "P-05C0-0_GA-1",
-                                "P-05C0-0_GA-2"
+                                {
+                                    "identifier": "P-05C0-0_GF-1",
+                                    "name": "",
+                                    "role": "SwitchOnOff",
```

Comment:
There are translation elements for this (for each FunctionPoint), but we should also save a common identifier (like the "SwitchOnOff" here, or the id `DR-1`) to be able to use that independently of language.

It should also have a `DatapointType` assigned - which is a valuable information.